src/server/game/Entities/Unit/Unit.h

```diff
@@ -23,6 +23,7 @@
 #include "EventProcessor.h"
 #include "FollowerReference.h"
 #include "FollowerRefManager.h"
+#include "GameTime.h"
```

Comment:
I will bitch about this in every single PR. DO NOT ADD USELESS INCLUDES TO HEADER FILES IF YOU CAN AVOID IT.
Move PlayerMovementPendingChange constructor to Unit.cpp